docs/api/rest_api/rest_api_reference/input/ez-types.raml

```diff
@@ -2128,401 +2140,1056 @@ View: #TODO outdated example
-    description: ''
+    properties:
+        identifier:
+             type: string
+        names:
+            type: array
+        descriptions:
+             type: array
+        global_price_rate:
+            type: numeric-string
 
 CustomerGroupUpdate:
+    description: 'This class is used to update a customer group.'
     type: object
-    description: ''
+    properties:
+        names:
+            type: array
+        identifier:
+             type: string
+        position:
+             type: integer
+        languages:
+            type: array
 
 CustomerGroupViewInput:
+    description: 'This class represents a customer group view input.'
     type: object
-    description: ''
+    properties:
+        CustomerGroupQuery:
+            type: object
 
 CustomerGroupView:
+    description: 'This class represents a customer group view.'
     type: object
+    properties:
+        CurrencyView:
+            type: CurrencyView
+        identifier:
+            type: string
+        CurrencyQuery:
+            type: object
+        Result:
+            type: object
 
 CurrencyViewInput:
+    description: 'This class represents a currency view input.'
     type: object
-    description: ''
+    properties:
+        CurrencyQuery:
+            type: object
 
 ProductType:
+    description: 'This class represents a product type.'
     type: object
-    description: ''
+    properties:
+        identifier:
+            type: string
+        name:
+            type: string
+        AttributeAssignmentList:
+            type: AttributeAssignment[]
 
 ProductTypeUsage:
+    description: 'This class represents a product type usage.'
     type: object
-    description: ''
+    properties:
+        isUsed:
+            type: boolean
 
 ProductTypeCreate:
+    description: 'This class is used to create a product type.'
     type: object
-    description:
+    properties:
+        ContentTypeCreateStruct:
+            type: object
+        main_language_code:
+            type: string
+        assigned_attributes:
+            type: object
+            properties:
+                identifier:
+                    type: string
+                is_required:
+                    type: boolean
+                is_discriminator:
+                    type: boolean
 
 ProductTypeUpdate:
+    description: 'This class is used to update a product type.'
     type: object
-    description:
+    properties:
+        ContentTypeUpdateStruct:
+            type: object
+        assigned_attributes:
+            type: object
+            properties:
+                identifier:
+                    type: string
+                is_required:
+                    type: boolean
+                is_discriminator:
+                    type: boolean
+
 
 ProductTypeViewInput:
+    description: 'This class represents a product type view input.'
     type: object
-    description: ''
+        properties:
+            ProductTypeQuery:
+                type: object
```

Comment:
`A colon cannot be used in an unquoted mapping value at line 2886 (near "            type: object").`

Indent error fix:
```suggestion
    properties:
        ProductTypeQuery:
            type: object
```